src/main/scala/net/snowflake/spark/snowflake/pushdowns/querygeneration/QueryBuilder.scala

```diff
@@ -39,6 +40,11 @@ private[querygeneration] class QueryBuilder(plan: LogicalPlan) {
     */
   private var foundSnowflakeRelation = false
 
+  /**
+    * Logger for pushdown processing
+    */
+  private[querygeneration] val log = LoggerFactory.getLogger(getClass)
```

Comment:
It is ok for now.
I think we should use Spark Logging instead of Logger. it is a big project, many codes should be refactored.

Reply:
I think the SC and spark already shared the same logging file.

Reply:
Yes, they do. I mean we can simplify the interface